frontend/src/components/RatingTable.tsx

```diff
@@ -117,10 +133,39 @@ export const ratingTable: (def: RatingTableDef) => FC<RatingTableProps> = ({
     );
   };
 
+  const onSortingChange = (sorter: TaskRatingSortingTypes) =>
+    sorting.type.set(sorter);
+
   return (
     <div className={classes(css.ratingContainer)}>
       <div className={classes(css.titleContainer)}>
         <h3>{titleCase(t(typeString))}</h3>
+        <div className={classes(css.rightSide)}>
+          <Trans i18nKey="List order" />
+          <Dropdown
+            css={css}
+            title={TaskRatingSortingTypesToText(sorting.type.get())}
+            maxLength={20}
+          >
+            {Object.values(TaskRatingSortingTypes)
+              .filter(
+                (sortingType): sortingType is TaskRatingSortingTypes =>
+                  typeof sortingType !== 'string',
+              )
+              .map((value) => (
+                <div
+                  className={classes(css.dropItem)}
+                  key={value}
+                  onClick={() => onSortingChange(value)}
+                  onKeyPress={() => onSortingChange(value)}
+                  role="button"
+                  tabIndex={0}
+                >
```

Comment:
Could change the `div` to an actual `button`.

This has at least couple advantages:
- `tabIndex` for free
- only needs to specify the selection action once (thus, I'd inline it)
- keyboard access
  - for free
  - more sane (only on <kbd>Space</kbd> and <kbd>Enter</kbd>, and closes on selection)

```suggestion
                <button
                  className={classes(css.dropItem)}
                  key={value}
                  onClick={() => sorting.type.set(value)}
                  type="button"
                >
```

Adding the following to `.dropItem` class seems to get the styles correct for buttons also:
```scss
width: 100%;
background-color: $COLOR_WHITE;
```